Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -41,91 +41,102 @@
 field = acts.ConstantBField(acts.Vector3(0.0, 0.0, 2.0 * u.T))
 rnd = acts.examples.RandomNumbers(seed=42)
 
-s = acts.examples.Sequencer(events=100, numThreads=-1, outputDir=str(outputDir))
+with acts.FpeMonitor():
```

Comment:
should we also do this to the ITk full chain?

Reply:
We don't run it in the CI though. But wouldn't hurt I guess?

Reply:
as you like. I think @timadye  wanted to keep them as close as possible